src/main/scala/nl.knaw.dans.easy.deposit/docs/DDM.scala

```diff
@@ -31,7 +30,10 @@ object DDM extends SchemedXml with DebugEnhancedLogging {
   override val schemaLocation: String = "https://easy.dans.knaw.nl/schemas/md/2018/05/ddm.xsd"
 
   def apply(dm: DatasetMetadata): Try[Elem] = Try {
-    val lang: String = dm.languageOfDescription.map(_.key).orNull // null omits attribute rendering
+    val lang: String = dm.languageOfDescription match {
+      case Some(SchemedKeyValue(_, Some(str), _)) if str.nonEmpty => str
```

Comment:
1. this sounds like `collect { ... }.orNull`
2. `str.nonEmpty` or `!str.isBlank` (from `dans-scala-lib`)?